API.md

```diff
@@ -0,0 +1,1819 @@
+            "text": "used-a-fever-reducer",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "4",
+            "text": "chills",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "5",
+            "text": "repeated-shaking-with-chills",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "6",
+            "text": "muscle-pain",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "7",
+            "text": "headache",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "8",
+            "text": "sore-throat",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "9",
+            "text": "new-loss-of-taste-or-smell",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          }
+        ],
+        "resourceType": "QuestionnaireResponse"
+      }
+    }
+  ],
+  "resourceType": "Bundle"
+}
+```
+</details>
+
+
+<a name="create"/>
+
+## Creating
+
+The API supports creating new monitorees.
+
+### Extensions
+
+<a name="create-ext"/>
+
+Along with supporting the US Core extensions for race, ethnicity, and birthsex, Sara Alert includes four additional extensions for things specific to the Sara Alert workflows.
+
+Use `http://saraalert.org/preferred_contact_method` for specifying the monitorees Sara Alert preferred contact method (options are: `E-mailed Web Link`, `SMS Texted Weblink`, `Telephone call`, and `SMS Text-message`).
+
+```json
+{
+  "url": "http://saraalert.org/preferred_contact_method",
+  "valueString": "E-mailed Web Link"
+}
+```
+
+Use `http://saraalert.org/symptom_onset_date` to specify when the monitoree's first symptoms appeared for use in the Sara Alert isolation workflow.
+
+```json
+{
+  "url": "http://saraalert.org/symptom_onset_date",
+  "valueDate": "2020-05-23"
+}
+```
+
+Use `http://saraalert.org/last_exposure_date` to specify when the monitoree's last exposure occurred for use in the Sara Alert exposure workflow.
+
+
+```json
+{
+  "url": "http://saraalert.org/last_exposure_date",
+  "valueDate": "2020-05-18"
+}
+```
+
+Use `http://saraalert.org/isolation` to specify if the monitoree should be in the isolation workflow (omitting this extension defaults this value to false, leaving the monitoree in the exposure workflow).
+
+```json
+{
+  "url": "http://saraalert.org/isolation",
+  "valueBoolean": false
+}
```

Comment:
This should probably be an Observation.

"Isolation or quarantine for active infectious disease in last 14D - while a resident" https://loinc.org/55017-8/

or

"Isolation or quarantine for active infectious disease in last 14D - while not a resident" https://loinc.org/55016-0/

may be relevant. Both are yes/no coded answers.